samples/guide/src/main/java/okhttp3/recipes/PostMultipart.java

```diff
@@ -40,7 +41,9 @@ public void run() throws Exception {
         .setType(MultipartBody.FORM)
         .addFormDataPart("title", "Square Logo")
         .addFormDataPart("image", "logo-square.png",
-            RequestBody.create(new File("website/static/logo-square.png"), MEDIA_TYPE_PNG))
+            RequestBody.create(
+                new File(SamplesKt.getPrefix() + "/docs/images/logo-square.png"),
```

Comment:
the path change is the only required fix, we can ditch the test and getPrefix() logic